db/seeds.rb

```diff
@@ -5,3 +5,17 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create!(email: "user@example.com", password: "password",
+  password_confirmation: "password")
+
+User.create!(email: "lunk@example.com", password: "password",
+  password_confirmation: "password")
+
+User.create!(email: "zorldo@example.com", password: "password",
+  password_confirmation: "password")
+
+
+200.times do
+  Post.create!(user_id: rand(1..3), content: Faker::Hipster.sentence)
```

Comment:
Buzzword is pretty good. The hardest part about faker is deciding which set to use. I used Faker::Hacker.say_something_smart all through the Iron Yard.